pos_cash_move_reason/tests/test_pos_cash_move_reason.py

```diff
@@ -22,6 +23,10 @@ def setUpClass(cls):
         )[0]
         cls.deposit_reason = cls.env.ref("pos_cash_move_reason.bank_out_reason")
 
+    def test_onchange_expense_reason(self):
+        self.deposit_reason._onchange_is_expense_reason()
```

Comment:
The way to test onchanges is to use a Form

Reply:
Hi @etobella 
I've updated the test for onchange using Form. You can review it.